addons/aws/1.0.1/kubeadm-cluster-config-v1beta2.yml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+metadata:
+  name: kubeadm-cluster-configuration
+clusterName: $KUBERNETES_CLUSTER_NAME
+networking:
+  podSubnet: $POD_CIDR
+  serviceSubnet: $SERVICE_CIDR
+apiServer:
+  extraArgs:
+    cloud-provider: aws
+    enable-admission-plugins: DefaultStorageClass
+controllerManager:
+  extraArgs:
+    cloud-provider: aws
+    configure-cloud-routes: '"true"'
```

Comment:
i don't believe we want this if we are using a CNI. you need to either remove this or make it optional